### PR TITLE
[EMB-176] Domain should always be hostname, not always localhost or .osf.io

### PR DESCRIPTION
**Pull request by hmoco**
Fix not clearing right cookie due to different domain.

### CODE REVIEW

File: app/services/status-messages.ts [hunk 1]
```diff
@@ -29,15 +29,15 @@ export default class StatusMessages extends Service {
...
+        status = status.slice(1, -1);
         let messages = JSON.parse(status);
         messages = messages.map(each => {
             const updated = each;
             updated.id = `status.${each.id}`;
             return updated;
         });
         cookies.clear(config.OSF.statusCookie, {
-            domain: config.environment === 'development' ? 'localhost' : '.osf.io',
+            domain: window.location.hostname,
```
Comment (binoculars) [inline app/services/status-messages.ts]: Needs to come from the config so that it’s overridable. Hostname won’t work either because we use .osf.io

Comment (hmoco) [reply app/services/status-messages.ts]: will put it into config! However, we don't use .osf.io for the cookies being created by flask response .set_cookie

**Comment by hmoco** [comment]
I am assuming this will need staging to pass in 'staging.osf.io' as OSF_COOKIE_DOMAIN and so on for other environments

File: app/components/file-browser/component.ts [hunk 2]
```diff
@@ -95,7 +95,7 @@ export default class FileBrowser extends Component.extend({
     isMoving = false;
     loaded = true;
     uploading = A([]);
-    clickable = ['.dz-upload-button'];
+    clickable = ['.dz-upload-button', '.file-browser-empty'];
```
Comment (binoculars) [inline app/components/file-browser/component.ts]: Probably going to get merge conflicts with #162

Comment (hmoco) [reply app/components/file-browser/component.ts]: Oh wow yeah I messed up my branches this should be here. Good catch @binoculars

File: app/components/file-browser/template.hbs [hunk 3]
```diff
@@ -301,16 +301,18 @@
                     </div>
                 {{/each}}
             {{else}}
-                <div class="file-placeholder">
-                    {{#if canEdit}}
+                {{#if canEdit}}
```
Comment (brianjgeiger) [inline app/components/file-browser/template.hbs]: Is this file related to the cookie domain somehow?
